### DISCUSSION TITLE
Free PC for testing

### DISCUSSION THREAD

**Discussion by dsf01** [jasonacox/Powerwall-Dashboard#303] [General]
I have a PC that someone can have for free if interested - just pay shipping (12# + packing).

PC only, no mouse/keyboard.  Uses standard AC plug (NEMA 5-10P), not included.

Dell Optiplex 390
250GB SSD / 4GB RAM
Intel core i3

I know this isn’t Craigslist, I just wanted to see if someone needed a test device.  Jason, feel free to delete if not applicable to this discussion.

**Comment by jasonacox** [comment]
That's very nice of you @dsf01 !  🙇  Where are you located (eg. continent) for shipping considerations?

Perfect system for someone wanting to setup a dedicated rig or test rig for their Dashboard and other containers... just add Ubuntu!

**Reply by dsf01** [reply]
> That's very nice of you @dsf01 ! 🙇 Where are you located (eg. continent) for shipping considerations?
> 
> Perfect system for someone wanting to setup a dedicated rig or test rig for their Dashboard and other containers... just add Ubuntu!

Ships from Irvine, California.
